docs/app-registration.md

```diff
@@ -3,17 +3,27 @@ id: app-registration
+As mentioned in the [How it Works](/how-it-works) section, app registration is _not_ required to get started with creating accounts and submitting simple payments. Developers can get started by configuring their SDKs to use the [Test Environment](/terms-and-concepts#test-environment).
 
-However, to qualify for KRE rewards, make fee-less transactions, or to make use of [webhooks](/how-it-works#webhooks) or [invoicing](/how-it-works#invoices) in the hosted version of Agora, apps must register using [this form](https://docs.google.com/forms/d/e/1FAIpQLSdz60FPmUB7qBq-TF7NNmRgM5W8wIqqL5oVHmMRbtBBXppv4Q/viewform). After submitting the form, an email will be sent within 24-48 hours with additional information for getting set up on the production and test environments. 
+However, to qualify for KRE rewards, make fee-less transactions, or to make use of [webhooks](/how-it-works#webhooks) or [invoicing](/how-it-works#invoices) in the hosted version of Agora, apps must register using the [Kin Developer Portal](https://portal.kin.org). After registering, you'll be able to create a new app, be automatically issued an App Index, and manage the production and test environments.
 
-Upon registration, apps will be issued an [app index](/terms-and-concepts#app-index) for their app, which should be included in the [memo](/how-it-works#kin-binary-memo-format) of transactions (or when using the [SDKs](/intro#available-sdks)) for transactions to be properly attributed to specific apps. When you initialize your Kin SDK with your app index, it automatically gets included in the memo of transactions sent by your users and/or backend server.
+After you've [registered for the Developer Portal and created an app](#register), your app will be issued an [app index](/terms-and-concepts#app-index), which should be included in the [memo](/how-it-works#kin-binary-memo-format) of transactions (or when using the [SDKs](/intro#available-sdks)) for transactions to be properly attributed to specific apps. When you initialize your Kin SDK with your app index, it automatically gets included in the memo of transactions sent by your users and/or backend server.
 
 ## Register
 
-To register, please submit a response using [this form](https://docs.google.com/forms/d/e/1FAIpQLSdz60FPmUB7qBq-TF7NNmRgM5W8wIqqL5oVHmMRbtBBXppv4Q/viewform).
+To register your app:
 
-Once the form has been submitted, an email will be sent within 24-48 hours with additional information regarding getting set up on the production and test environments.
+- Create an account on the [Kin Developer Portal](https://portal.kin.org)
+- We'll send an email to the account you used during registration. Open that email and follow the instructions to verify your account
+- Login to the [Kin Developer Portal](https://portal.kin.org) and navigate to Apps
+- Click "Create App", fill in your details, and submit
```

Comment:
If we like, we can link to the "Create App" form directly using the URL https://portal.kin.org/apps/create